server/planning/events/events_template.py

```diff
@@ -94,7 +95,7 @@ class EventsTemplateResource(Resource):
             embeddable=False,
             required=True,
         ),
-        "data": {"type": "dict", "schema": _event_fields},
+        "data": {"type": "dict", "schema": _event_fields, "allow_unknown": True},
```

Comment:
`schema` can be an empty dict, and we can delete the `_event_field` attribute on the class, otherwise the underlying framework (Eve) will validate the `data` field against this schema (which does not add any value here)